File: Java/Miscellaneous/PrintAtHomeTicketInfo.java
package LTD_API;

import java.util.List;

public class PrintAtHomeTicketInfo
{
    private int? ticketId;
	private String ticketName1;
	private String ticketName2;
	
	public int? getTicketId()
    {
        return ticketId;
    }
    public void setTicketId(int? ticketId)
    {
        this.ticketId = ticketId;
    }

	public String getTicketName1()
    {
        return ticketName1;
    }
    public void setTicketName1(String ticketName1)
    {
        this.ticketName1 = ticketName1;
    }

	public String getTicketName2()
    {
        return ticketName2;
    }
    public void setTicketName2(String ticketName2)
    {
        this.ticketName2 = ticketName2;
    }
}

File: PHP/Miscellaneous/PrintAtHomeTicketInfo.php
<?php
namespace LTD_API;


class PrintAtHomeTicketInfo
{
    //instance of int?
	private $ticketId;
	private $ticketName1;
	private $ticketName2;
	
	public function getTicketId()
    {
        return $this->ticketId;
    }
    public function setTicketId($ticketId)
    {
        $this->ticketId = $ticketId;
    }

	public function getTicketName1()
    {
        return $this->ticketName1;
    }
    public function setTicketName1($ticketName1)
    {
        $this->ticketName1 = $ticketName1;
    }

	public function getTicketName2()
    {
        return $this->ticketName2;
    }
    public function setTicketName2($ticketName2)
    {
        $this->ticketName2 = $ticketName2;
    }
}

